apps/website/src/components/Playground/docusaurusPlugin.js

```diff
@@ -0,0 +1,23 @@
+const path = require('path');
+
+// Docusaurus plugin that modifies Webpack config to process files in "templates" folder.
+/** @type {import('@docusaurus/types').PluginModule} */
+module.exports = function () {
+  return {
+    name: 'playground-docusaurus-plugin',
+    configureWebpack() {
+      console.log(path.resolve(__dirname, 'code', 'templates'));
+      return {
+        module: {
+          rules: [
+            {
+              test: /\.js$/,
+              include: path.resolve(__dirname, 'code', 'templates'),
+              use: [{ loader: 'raw-loader' }, { loader: require.resolve('./webpackLoader') }],
+            },
```

Comment:
This tweak adds custom Webpack loader that removes `meta` exports that are visible in playground templates:

![image](https://user-images.githubusercontent.com/14183168/179706256-045d8c44-85d4-49d6-a28c-d637b97b2333.png)